data/json/recipes/recipe_ammo.json

```diff
@@ -117,11 +117,11 @@
   "reversible": false,
   "autolearn": true,
   "book_learn": [[ "glassblowing_book", 5 ]],
+  "qualities":[
+    {"id":"GRIP","level":3}
```

Comment:
Well I did work through and merge #18536. The other PR is a merge conflict with someone elses branch and there is no advantage over manual editing vs automated formatting - the latter is backed by unit tests to prevent problems.